src/gru/gru.py

```diff
@@ -21,7 +21,8 @@ def gru():
     not_running_workflows = [
         action_name
         for action_name in tests["action_name"]
-        if action_name in workflows and workflows[action_name]["status"] not in ["queued", "in_progress"]
+        if action_name in workflows
+        and workflows[action_name]["status"] not in ["queued", "in_progress"]
```

Comment:
Hmm this was `black` but I don't love the and being at the same indent. Fine just going with black's suggestion though

Reply:
yea it hurts visually, but agree with using their rules (unless we explicitly exclude it in a configuration file)

flagging though, I've seen unexpected behavior with conditional statements like `x and y` when either are more complicated than a single variable. reminds of PEMDAS. this _might_ evaluate as:
```python
if ((action_name in workflows) and workflows[action_name]["status"]) not in ["queued", "in_progress"]
```

if this is working as expected, no worries! but it may be worth double confirming they are, and if they aren't, something like `... and (workflows[action_name]["status"] not in ["queued", "in_progress"])` might fix it

Reply:
Oh good call, I think I've run into that before in other cases, and it's def possible I've just sort of lucked into it working so far based on the data coming in.